frontend/src/document/document.html

```diff
@@ -71,5 +71,28 @@
         {{path.value}}
       </div>
     </div>
+    <modal v-if="shouldShowConfirmModal">
+      <template v-slot:body>
+        <div class="modal-exit" @click="shouldShowConfirmModal = false;">&times;</div>
+        <h2>
+          Are you sure you want to save the following changes?
+        </h2>
+        <div v-for="change in Object.keys(changes)">
```

Comment:
Lets instead display `changes` using Prism, rather than a separate div for each key